crates/flake8_to_ruff/src/converter.rs

```diff
@@ -117,8 +117,8 @@ pub(crate) fn convert(
                 "builtins" => {
                     options.builtins = Some(parser::parse_strings(value.as_ref()));
                 }
-                "max-line-length" | "max_line_length" => match LineLength::from_str(value) {
-                    Ok(line_length) => options.line_length = Some(line_length),
+                "max-line-length" | "max_line_length" => match LineWidth::from_str(value) {
+                    Ok(line_length) => options.line_width = Some(line_length),
```

Comment:
I'm assuming that the `line_length` here wasn't renamed because it's coming from flake8?

Reply:
Yes, these are the options of the source configuration that gets rewritten to ruff.